CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] OTLP: Add support for converting OTel Exponential Histograms to Prometheus Native Histograms. #4063
+* [BUGFIX] OTLP: Do not drop exemplars of the OTLP Monotonic Sum metric. #4063
```

Comment:
Please move entries to their relevant sections (order is: CHANGE, FEATURE, ENHANCEMENT, BUGFIX)